src/lipo.rs

```diff
@@ -5,6 +5,18 @@ use failure::ResultExt;
 use log::info;
 use std::fs;
 use std::process::Command;
+use std::path::Path;
+
+fn is_output_updated(output: impl AsRef<Path>, inputs: impl IntoIterator<Item=impl AsRef<Path>>) -> std::io::Result<bool> {
+    let output_mtime = fs::metadata(output)?.modified()?;
```

Comment:
Can we handle the "file not found" case here explicitly (by returning that the output needs to be updated)? Then we can also log a warning below if this function fails.